KIPs/kip-draft_kgp6.md

```diff
@@ -0,0 +1,672 @@
+---
+kip: 103
+title: Treasury Fund Rebalancing
+author: Aidan<aidan.kwon@klaytn.foundation>, Toniya<toniya.sun@klaytn.foundation>
```

Comment:
```suggestion
author: Aidan (@aidan-kwon), Toniya (@toniya-klaytn)
```

It could be better to use github id instead of email. As we all know it is very high chance to change your email compared to github id.